src/components/search/History/SearchHistory.scss

```diff
@@ -0,0 +1,37 @@
+@import "src/styles/scaffold";
+
```

Comment:
This component's structure and styles are very similar to the [Top Search component](https://github.com/vtex-sites/base.store/pull/355/files#diff-7705543ff64105143829013d8db01ea8497c88b73b86b9050f8b1bc1e3a52aafR2). At some point we can try to unify them to reduce some duplication.

Reply:
Agreed.